project/dexter-core/src/test/com/samsung/sec/dexter/core/TestJavaFile_For_TargetFileTest.java

```diff
@@ -0,0 +1,4 @@
+package com.samsung.sec.dexter.core;
```

Comment:
It is used for different tests that should NOT be a part of this pull request. I will delete it for now.